libusb/os/windows_winusb.c

```diff
@@ -898,6 +900,7 @@ static int init_device(struct libusb_device *dev, struct libusb_device *parent_d
 		case 1: dev->speed = LIBUSB_SPEED_FULL; break;
 		case 2: dev->speed = LIBUSB_SPEED_HIGH; break;
 		case 3: dev->speed = LIBUSB_SPEED_SUPER; break;
+		case 4: dev->speed = LIBUSB_SPEED_SUPER_PLUS; break;
```

Comment:
There are some checks in the code that checks for speed == LIBUSB_SPEED_SUPER. You might want to make sure that they also check for LIBUSB_SPEED_SUPER where this applies.

Reply:
I know that libusb_get_max_iso_packet_size is doing it for instance. There might be other places.

Reply:
PS: l am not a maintainer, so you might want to wait to get a feedback from them.

Reply:
I have added that as a separate possible bug. It is not caused by this change. https://github.com/libusb/libusb/issues/553

Reply:
It's probably just a matter of changing the == to >= if the spec for isochronous packet size has not changed in SuperSpeed Plus (I don't know if it has).